python/lsst/verify/gen2tasks/metricTask.py

```diff
@@ -26,16 +26,15 @@
 
 from deprecated.sphinx import deprecated
 
-import lsst.pipe.base as pipeBase
+from ..tasks.metricTask import MetricTask as NewTask
```

Comment:
Could you use the full `lsst.verify.tasks.MetricTask` here?

Also, could you come up with a different name other than `NewTask`? `Gen3MetricTask` would be good if that is correct, but otherwise even `NewMetricTask` would be okay.